ui/src/shared/components/TimeSeries.tsx

```diff
@@ -189,7 +200,11 @@ class TimeSeries extends Component<Props & WithRouterProps, State> {
       let statuses = [] as StatusRow[][]
       if (check) {
         const extern = buildVarsOption(variables)
-        this.pendingCheckStatuses = runStatusesQuery(orgID, check.id, extern)
+        this.pendingCheckStatuses = runStatusesQuery(
+          this.props.params.orgID,
```

Comment:
is this supposed to be `orgID` from line 188?

Reply:
arghh.. soooooooo....... statuses are always in this orgs _monitoring bucket. so, no... But I realize this is very potentially confusing down the line